webextensions/api/pageAction.json

```diff
@@ -35,7 +35,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction/ImageDataType",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "24"
```

Comment:
❌ https://github.com/chromium/chromium/commit/866d940a3eb21ae92ef7866c9155b936fd6ed917 landed in [23.0.1263.0](https://github.com/chromium/chromium/releases/tag/23.0.1263.0) before 23.0.1271.

```suggestion
                "version_added": "23"
```